docs/dev-tools/console/console.asciidoc

```diff
@@ -46,6 +46,8 @@ curl -XGET "http://localhost:9200/_search" -d'
 When you paste the command into Console, {kib} automatically converts it
 to Console syntax.  Alternatively, if you want to see Console syntax in cURL, 
 click the action icon (image:dev-tools/console/images/wrench.png[]) and select *Copy as cURL*.
+Please note that once copied, the url will need to be replaced and the username and password will 
```

Comment:
I'm not sure you need to include "the url will need to be replaced". Can you elaborate on what you mean by that? Console should construct the url to include the correct host, based on configuration of `elasticsearch.hosts` in your `kibana.yml`.

Reply:
@hp0620 Thank you for your contribution.  Once you address the comment above, we'll be able to merge your change.